roles/heat/templates/etc/heat/heat.conf

```diff
@@ -55,6 +55,7 @@ num_engine_workers = {{ heat.engine_workers }}
 
 [database]
 connection=mysql://heat:{{ secrets.db_password }}@{{ endpoints.db }}/heat?charset=utf8
+max_overflow=1000
```

Comment:
I can't think of a reason we'll want to adjust this up/down really. We're just going to set a high mark, until the code does it automagically. That work will eventually backport into Kilo, or we'll upgrade to a codebase that has it and then we'll remove this line all together.